### ISSUE TITLE
Error with hygraph endpoint

### ISSUE THREAD

**Issue by samburgers** [remorses/genql#122]
Getting this error from https://hygraph.com/ endpoint when making queries

```
./src/api/generated/schema.ts
Error: 
  x the name `enumSystemDateTimeFieldVariation` is defined multiple times
      ,-[~/src/api/generated/schema.ts:6431:1]
 6431 |    PUBLISHED: 'PUBLISHED' as const
 6432 | }
 6433 | 
 6434 | export const enumSystemDateTimeFieldVariation = {
      :              ^^^^^^^^^^^^^^^^|^^^^^^^^^^^^^^^
      :                              `-- previous definition of `enumSystemDateTimeFieldVariation` here
 6435 |    BASE: 'BASE' as const,
 6436 |    LOCALIZATION: 'LOCALIZATION' as const,
 6437 |    COMBINED: 'COMBINED' as const
 6438 | }
 ```
 
 the generated `schema.ts` seems to have both of these
 
 ```
 export const enumSystemDateTimeFieldVariation = {
   BASE: 'BASE' as const,
   LOCALIZATION: 'LOCALIZATION' as const,
   COMBINED: 'COMBINED' as const
}

...

export const enumSystemDateTimeFieldVariation = {
   base: 'base' as const,
   localization: 'localization' as const,
   combined: 'combined' as const
}
```
 
Thanks in advance!

**Comment by samburgers** [comment]
<img width="396" alt="Screenshot 2023-05-06 at 1 57 34 pm" src="https://user-images.githubusercontent.com/1476939/236598780-5f13bf47-3a7b-441b-966b-261388d00429.png">

fwiw this is what the graphql explorer shows, each one has the same properties but with different casing (similar to above)

**Comment by remorses** [comment]
Ok, thank you for reporting this, will be fixed today

**Comment by remorses** [comment]
Fixed in 5.0.1

**Comment by samburgers** [comment]
Amazing, confirming that fixes it. Thank you!
